__tests__/wasmClient.test.ts

```diff
@@ -0,0 +1,10 @@
+import { createRunWasmClient } from '../src/index'
+import { RunWasmClient } from '../src/RunWasmClient'
+
+describe('createRunWasmClient', () => {
+  it('should return an instance of WasmClient', () => {
+    const wasmClient = createRunWasmClient('python')
+
+    expect(wasmClient).toBeInstanceOf(RunWasmClient)
+  })
+})
```

Comment:
👍🏼 Very very basic test to make the function returns what it says it's going to return